test/helper/randomNumberGenerator.test.js

```diff
@@ -0,0 +1,36 @@
+const { getRandomInt } = require('../../src/helper/randomNumberGenerator');
+const { expect, assert } = require('chai');
+
+describe('randomNumberGenerator helper', () => {
+	it('should throw an exception when difference between max number and min number exceed allowed treshold 256^6-1 (6 bytes)', () => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 135. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5483065)